script/convert_tags_to_topic.rb

```diff
@@ -40,7 +51,7 @@ def put_things_in_tag_to_correct_position cards, skip_year
     card.type_id = Card::WikirateTopicID
     puts "Updating #{card.name}'s type to topic".green
     card.save!
-    Card.cache.reset_soft
+    Card.cache.reset
```

Comment:
I remember you asked me to add this when we were desperate with the performance.